stdlib/benford.hk

```diff
@@ -0,0 +1,5 @@
+#Implementation of the benford distribution
+
+def benford():
```

Comment:
Again, this doesn't need to be in a file of it's own. It fits better with uniformTransforms. Y'know, since it's implemented as a transformation on the uniform distribution